backend/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src"],
+  "include": ["src", "src/helpers/mailtrap/mailtrap.ts"],
```

Comment:
Why do we need to explicitly include smth into types as it's already in the `src` folder?